staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/metrics_test.go

```diff
@@ -19,36 +19,27 @@ package exec
 import (
 	"testing"
 	"time"
-
-	"k8s.io/client-go/tools/metrics"
 )
 
-type mockTTLGauge struct {
-	v *time.Duration
+type mockExpiryGauge struct {
+	v *time.Time
 }
 
-func (m *mockTTLGauge) Set(d *time.Duration) {
-	m.v = d
+func (m *mockExpiryGauge) Set(t *time.Time) {
+	m.v = t
 }
 
-func ptr(d time.Duration) *time.Duration {
+func ptr(d time.Time) *time.Time {
```

Comment:
nit: argument should be `t` now probably

Reply:
Done.